internal/install/static_snapshot_yml.go

```diff
@@ -91,6 +90,7 @@ services:
     - "FLEET_ENROLL=1"
     - "FLEET_ENROLL_INSECURE=1"
     - "FLEET_SETUP=1"
+    - "FLEET_URL=http://kibana:5601"
```

Comment:
@blakerouse Is this required for the case where fleet-server runs as part of the Elastic Agent?

Reply:
With `FLEET_SERVER_ENABLE` set to true, `FLEET_URL` is ignored. `FLEET_URL` is required when `FLEET_SERVER_ENABLE` is not set.